scheduler-driver/src/main/java/gov/nasa/jpl/aerie/scheduler/EquationSolvingAlgorithms.java

```diff
@@ -127,6 +127,9 @@ private IteratingResult nextValueAt(
         final int maxIteration)
     throws ExceededMaxIterationException, SchedulingInterruptedException
     {
+      // the number of possible values may be less than the number of iterations, so stop after all have been visited.
+      long numTimepoints = max.in(Duration.MICROSECONDS) - min.in(Duration.MICROSECONDS) - 1;
```

Comment:
this is a numeric overflow if `max` is `Long.MAX_VALUE` and `min` is `Long.MIN_VALUE`.

Reply:
maybe use `BigInteger` ?

Reply:
I pasted in some code for handling overflow.